docs/cli/convert.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Convert
+description: Convert SQL migration files to a pgroll migration
+---
+
+## Command
+
+```
+$ pgroll convert --name create_first_table /path/to/migration.sql
+```
+
+This reads the SQL statements from `/path/to/migration.sql` and translates them into a `pgroll` migration.
+The name of the migration is going to be `create_first_table`.
```

Comment:
I think we shouldn't use hard line breaks in `mdx`? I'd put this on the line above.